esti/s3_gateway_test.go

```diff
@@ -289,6 +289,121 @@ func TestS3IfNoneMatch(t *testing.T) {
 	}
 }
 
+func TestListMultipartUploads(t *testing.T) {
+	blockStoreType := viper.GetString(ViperBlockstoreType)
+	if blockStoreType != "s3" {
+		t.Skip("Skipping test - blockstore type is not s3")
+	}
```

Comment:
I would check that we return the correct error for other blockstores (`NotImplemented`)